src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -16,6 +16,7 @@
  */
 package com.g4s8.ghman.bot;
```

Comment:
@scristalli I have not mentioned `TkBot` in #45, but it would be better remove data source dependancy from it too. Thus it will be easier to unit test this class.

Reply:
@olenagerasimova done in commit 321d4a8

Reply:
@g4s8 a question: for changes like this one, is `ARC` confirmation required? If so, where? Here in the PR, or in the original issue? Should the issue description be edited now, if this change is approved?

Reply:
@olenagerasimova @scristalli it'd be better to make this decisions right in the origin ticket and update ticket description before submitting PR.

Reply:
@g4s8 thank you, I'll make sure I follow this procedure in the future